### PR TITLE
Create a tea bag

### PR DESCRIPTION
**Pull request by ouafaema**
Scrum-35 : As a MINTER, I can create a TEA BAG

### CODE REVIEW

**Comment by sckaper** [comment]
link your code and fix the build first please

**Review by fourm1 — CHANGES_REQUESTED** [review]
Delete all the commentaries

**Review by fourm1 — APPROVED** [review]
This seems good now

**Review by sckaper — APPROVED** [review]
that's functionnal

File: app/api/profile/update/route.ts [hunk 1]
```diff
@@ -14,12 +14,12 @@ import {NextResponse} from "next/server"
 
 
 export const POST = auth(async (req: NextAuthRequest) => {
-  const formData = await req.formData()
-  const username = formData.get("username") as string
-  const bio = formData.get("bio") as string
-  const link = formData.get("link") as string
-  const imageFile = formData.get("avatar") as string
-
+  //Const formData = await req.formData()
```
Comment (avetisk) [inline app/api/profile/update/route.ts:17]: Dead code.

File: app/api/tea-bag/get/route.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import {auth, getSession} from "@/auth"
+// @ts-expect-error TODO fix library not found
```
Comment (avetisk) [inline app/api/tea-bag/get/route.ts:2]: You should add the TODO in your issue tracker, not here.

File: app/api/tea-bag/route.ts [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  if (!username || !bio || !link) {
+    return problem({...invalidQueryParameterProblem, detail: "username, bio, and link are required"})
+  }
+
+  const session = getSession(req)
+
+  if (!session) {
+    return problem({...notAuthenticatedProblem, detail: "You need to be authenticated to create this profile"})
+  }
+
```
Comment (avetisk) [inline app/api/tea-bag/route.ts:70]: Use a lib like `zod` or `yup`.

File: app/profile/[slug]/ssr.tsx [hunk 4]
```diff
@@ -97,6 +106,76 @@ export function ProfileContainer({link, avatarUrl, username}: ProfileContainerPr
...
+        <div onClick={() => {
+          onReport(id as number)
+        }}>
+          <ReportIcon className="size-6" color="red"/>
+        </div>
+
+        <div onClick={() => {
+          onDelete(id as number)
+        }}>
+          <TrashIcon className="size-6" color="red"/>
```
Comment (avetisk) [inline app/profile/[slug]/ssr.tsx:172]: Declare your handlers before using them in JSX.

**Pull request merged**